apis/v1alpha1/providerconfig_types.go

```diff
@@ -35,6 +35,14 @@ type ProviderConfigSpec struct {
 	// +optional
 	Insecure *bool `json:"insecure,omitempty"`
 
+	// Enables gRPC-web protocol. Useful if Argo CD server is behind proxy which does not support HTTP2.
+	// +optional
+	GRPCWeb         *bool `json:"grpcWeb,omitempty"`
+
+	// Enables gRPC-web protocol. Useful if Argo CD server is behind proxy which does not support HTTP2. Set web root.
+	// +optional
+	GRPCWebRootPath string `json:"grpcWebRootPath,omitempty"`
```

Comment:
Optional fields should be a pointer

```suggestion
	GRPCWebRootPath *string `json:"grpcWebRootPath,omitempty"`
```